R/Viewer.R

```diff
@@ -286,7 +316,13 @@ on_connection_opened <- function(connection, code) {
 
   # use the database name as the display name
   display_name <- connection@info$dbname
```

Comment:
I think all the following can be
```r
server_name <- paste(collapse = "@", string_values(user_name, server_name))

if (!identical(server_name, display_name)) {
  display_name <- paste(collapse = " - ", string_values(display_name, server_name)
}
```

And possibly just put in a `computeDisplayName()` function.